relay-server/src/processing/spans/mod.rs

```diff
@@ -79,7 +81,10 @@ impl processing::Processor for SpansProcessor {
             .take_items_by(|item| matches!(*item.ty(), ItemType::Span))
             .into_vec();
 
-        let work = SerializedSpans { _headers, spans };
+        let work = SerializedSpans {
+            headers: _headers,
```

Comment:
I think the local should just be renamed to `headers`.
```suggestion
            headers,
```